batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/constants/Page.java

```diff
@@ -0,0 +1,20 @@
+package com.lftechnology.batch7crud.constants;
+
+import java.io.File;
+
+/**
+ * Created by sanjay on 1/21/16.
+ */
+public class Page {
+  private static final String SEPARATOR = File.separator;
+  private static final String BASE = SEPARATOR+"WEB-INF" + SEPARATOR + "views" +SEPARATOR;
+  public static final String EDIT = BASE + "edit.jsp";
+  public static final String DETAIL_VIEW = BASE + "detail-view.jsp";
+  public static final String LIST_VIEW = BASE + "list-view.jsp";
+  public static final String NEW_STUDENT = BASE + "newStudent.jsp";
+  public static final String INDEX = BASE + "index.jsp";
```

Comment:
I would prefer variable name to be INDEX_PAGE .. makes more sense and it is self explanatory too.